.github/workflows/deploy-prod.yaml

```diff
@@ -0,0 +1,63 @@
+name: Deploy Prod
+
+on:
+  release:
+    types: [published, created, edited]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  EC2-Deploy:
+    runs-on: ubuntu-latest
+    environment:
+      name: ${{ github.ref_name }}
```

Comment:
do we want this to be `main` (as is) or `PROD` or `timeline-report.bitovi-jira.com`?